android/app/src/main/java/com/dismoi/scout/accessibility/BootUpReceiver.kt

```diff
@@ -0,0 +1,20 @@
+package com.dismoi.scout.accessibility
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+
+class BootUpReceiver : BroadcastReceiver() {
+  override fun onReceive(context: Context, intent: Intent) {
+    if (intent.action === Intent.ACTION_BOOT_COMPLETED) {
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Same remark as before, and maybe there could be boolean constant telling if the feature is available instead of  having the test at many places.

Reply:
I can also put a decorator here: 

`@RequiresApi(Build.VERSION_CODES.O)`

Would you prefer?

Reply:
Wow looks neat!

Reply:
Ok created a new issue https://github.com/dis-moi/mobile_poc/issues/83